tests/py2puml/test_py2puml.py

```diff
@@ -15,26 +14,24 @@ def test_py2puml_model_on_py2uml_domain():
         puml_content = list(py2puml.py2puml('py2puml/domain', 'py2puml.domain'))
         for line_index, (actual_line, expected_line) in enumerate(zip(puml_content, expected_puml_file)):
             assert actual_line == expected_line, f'updated and versionned content {domain_diagram_file_path} in line {line_index} has changed'
-        
+
         assert line_index + 1 == len(puml_content), f'actual and expected diagrams have {line_index + 1} lines'
 
 
 def test_py2puml_with_subdomain():
-    expected = """@startuml
-class tests.modules.withsubdomain.subdomain.insubdomain.Engine {
-  horsepower: int
-}
-class tests.modules.withsubdomain.subdomain.insubdomain.Pilot {
-  name: str
-}
-class tests.modules.withsubdomain.withsubdomain.Car {
-  name: str
-  engine: Engine
-}
-tests.modules.withsubdomain.withsubdomain.Car *-- tests.modules.withsubdomain.subdomain.insubdomain.Engine
-@enduml
-"""
+    with open(DATA_DIR / 'with_subdomain.puml', 'r', encoding='utf-8') as fref:
+        expected_data = fref.read()
     puml_content = py2puml.py2puml(
       'tests/modules/withsubdomain/', 'tests.modules.withsubdomain'
     )
-    assert ''.join(puml_content) == expected
+    actual_data = ''.join(puml_content)
+    assert actual_data == expected_data
+
+
+def test_py2puml_with_methods():
+    """ Test py2puml with a class containing methods """
+    with open(DATA_DIR / 'with_methods.puml', 'r', encoding='utf-8') as fref:
+        expected_data = fref.read()
+    puml_content = py2puml.py2puml('tests/modules/withmethods', 'tests.modules.withmethods')
+    actual_data = ''.join(puml_content)
+    assert actual_data == expected_data
```

Comment:
in the main branch, there are now functions dedicated to comparing the output of a parsing command with the expected PlantUML contents. We should use them after rebasing.

Reply:
Agreed